manifests/server.pp

```diff
@@ -32,6 +32,7 @@
   Optional[Stdlib::Absolutepath] $ldapi_socket_path = undef,
   Optional[Boolean] $register_slp                   = undef,
   Optional[Stdlib::Absolutepath] $krb5_keytab_file  = undef,
+  Optional[Stdlib::Absolutepath] $krb5_client_keytab_file  = undef,
```

Comment:
please add a puppet-strings style documentation at the top (like it happened for manage_policy_rc_d)